notebooks/basics/quantum-circuits.ipynb

```diff
@@ -22,30 +22,43 @@
+    }
+   },
    "cell_type": "markdown",
    "id": "adfa4cb5",
    "metadata": {},
    "source": [
     "### 1.1 Boolean circuits\n",
     "\n",
-    "Here is an example of a (classical) Boolean circuit, where the wires carry binary values and the gates represent Boolean logic operations:\n",
+    "To illustrate these ideas, let's begin with a simple example. A *Boolean circuit* is one in which the output is a single classical bit (though the input could be $n$ bits). Here is an example:\n",
     "\n",
     "![Example of a Boolean circuit](images/quantum-circuits/Boolean-circuit-XOR.png)\n",
     "\n",
-    "The flow of information along the wires goes from left to right: the wires on the left-hand side of the figure labeled $\\mathsf{X}$ and $\\mathsf{Y}$ are input bits, which can each be set to whatever binary value we choose, and the wire on the right-hand side is the output.\n",
-    "The intermediate wires take whatever values are determined by the gates, which are evaluated from left to right.\n",
+    "The wires, represented by lines, carry binary values, i.e., bits of information. The gates, represented by labelled squares, are *Boolean logic operations*. These are operations which, given one or more input wires, output a single wire. As with circuits, the designation \"Boolean\" indicates a single binary output, but more general logic operations, with multiple output wires, are also possible. \n",
+    "\n",
+    "The flow of information along the wires goes from left to right. The wires on the leftmost side of the figure, labeled $\\mathsf{X}$ and $\\mathsf{Y}$, are the input bits, which can each be set to whatever binary value we choose. The wire on the right-hand side is the output, which is completely determined by the input and cannot be specified separately. Traveling along any wire, the value (0 or 1) stays the same, only changing when passing through a gate. Wherever two wires intersect, we use a filled-in dot to denote the wires being *connected*. If two wires cross with no dot, then the wires simply pass over one another, not affecting each other in any way. Other conventions exist, and sometimes a gap or a small loop is used to indicate unconnected crossing wires.\n",
     "\n",
-    "The gates are AND gates (labeled $\\wedge$), OR gates (labeled $\\vee$), and NOT gates (labeled $\\neg$).\n",
-    "The functions computed by these gates will likely be familiar to many readers, but here they are represented by tables of values:\n",
+    "![UnconnectedWire.png](attachment:UnconnectedWire.png)\n",
```

Comment:
Did you add this image? If so, can you put it in the `notebooks/basics/images/quantum-circuits` folder, and add it to the notebook using
```markdown
![description of the image](images/quantum-circuits/UnconnectedWire.png)
```

Reply:
Done! Thanks very much.